### DISCUSSION TITLE
Windows Build doesn't open on different Systems

### DISCUSSION THREAD

**Discussion by jonassorgenfrei** [AcademySoftwareFoundation/OpenRV#144] [Q&A]
Hey 

i sucessfully build OpenRV and everything's working fine on my machine and different machines of me. 

Unfortunalty copying the folder to some other windows 10 machienes is resulting in the typical terminal log but no Window opening. And the terminal process just finishes. Unfortunatly there was no error log in the command line. 
Is there anything i can do to debug this? Might be a problem with QT not being compatibal with the system i guess but not sure. 

Cheers
Jonas

**Comment by rogernelson** [comment]
Hi Jonas! Can you try launching RV directly from an mingw94 terminal? It will likely spit out some errors to the shell to indicate what is wrong.  If you don't have mingw on that system, it might also do the same if you just launch if from `cmd` or powershell.

**Reply by jonassorgenfrei** [reply]
Hey @rogernelson , 

Thanks for your answer.
yeah thats what i tried unfortunatly it just prints:

`C:\Users\user\Documents\OpenRV\bin>rv.exe
INFO: Using 'C:\Users\user' for $HOME.
rv.exe
Version 1.0.0, built on Feb  3 2023 at 00:52:05 (HEAD=9c6adc2).
Copyright (c) 2022 Autodesk, Inc. All rights reserved.

C:\Users\user\Documents\OpenRV\bin>`

And no actual error message. 
Is there may be some other place where it stores some log?

Cheers 
Jonas

**Reply by rogernelson** [reply]
If it crashes before it can even log anything, that is usually an indication of a segfault (ie the application accessed memory it wasn't supposed to). Unfortunately, the only real way to get to the bottom of what is happening in these cases is to attach a debugger to the application.  You'll also have to compile the application in debug mode (if you didn't do that initially). If you're using the rvcmds file to build the commands that can be done by postfixing a 'd' on to any one of the commands (`rvmkd` for example).

I wish I could give you an exact recipe to follow on windows, but I do all my debugging on Mac and Linux so I am a bit unfamiliar with how its done on Windows.  Depending on what you have installed on your system, this might be helpful:
https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/debugger-download-tools

If you are able to reproduce the crash in the debugger and print out a stack trace, that would give me a pretty good idea of what is happening.

**Reply by jonassorgenfrei** [reply]
Hey @rogernelson  

thanks for your suggestion. 
Will try building a debug build hopening for more output thanks!

Having a look on the windows Event Log shows the Application Error.
With the Exception Code 0xc0000005 

Which seems to say theres a .dll issue 

Additionally there was a Windows Error Reporting coming along:
Fault bucket, type 0 
Problem SIgnature:
P1: rv.exe
P2: 0.0.0.0
P3: 645278ae
P4: StackHash_ac46
P5: 0.0.0.0
P6: 00000000
P7: PCH_63_FROM_ntdll+0x000000000009DBB4
P8: c0000005
P9: 0000000000000008
P10:

But as always with windows kind of cryptically. Will try to compile with DEBUG mode to see if i can get more machienes.

A thing to mention again i compiled it with a different machien (same windows version tho!) where it works perfectly aswell as on most other machienes tho.  Only getting the issue on distribution. '

Cheers
Jonas

**Reply by jonassorgenfrei** [reply]
Looks liek the issue described here too: #94

**Comment by rogernelson** [comment]
It is possible it also due to a missing dll that exists on your machine but not the one you are trying to run it on. You could also try building on the machine where you are having the issue to see if you run into a missing dependency problem there.  Also, depending on how the other machine is configured it may be set up to block unsigned applications. Since you build it yourself (and I imagine you did not sign the binary after), the Windows security policy might be blocking it from running.  When we distribute the commercial RV we sign it with an Autodesk certificate to certify that the application comes from Autodesk, but that does not happen for OpenRV (which makes sense because Autodesk does not own OpenRV). These are just guesses on my part.

**Reply by jonassorgenfrei** [reply]
Cool thanks for your suggestions. Will try them. ;)
Actually just found out that RV doesn´t open aswell ont the machine, showing the message: 
Failed to load libEGL
Failed to load opengl32w.dll
Failed to load and resolve WGL/OpenGL functions. 
Guess this is a good hint will check the graphics drivers :)

Will keep you update if i finde anything.
Thanks!

Cheers 
Jonas
